lib/landrush/util/dnsmasq.rb

```diff
@@ -0,0 +1,10 @@
+module Landrush
+  module Util
+    class Dnsmasq
+      def self.nm_managed?(opts=nil)
+        nm_config = Pathname('/etc/NetworkManager/NetworkManager.conf')
+        File.exists?(nm_config) && File.readlines(nm_config).grep(/^dns=dnsmasq$/).any?
```

Comment:
I could probably use the inifile rubygem to do a little better job parsing the file to ensure the line is in the main section, but not sure how you feel about adding another dependency for the sake of improving a single config check.